base/metrics/metrics.go

```diff
@@ -15,10 +15,23 @@ var (
 	}, []string{"type"})
 )
 
+var (
```

Comment:
We can join these two `var` sections into one.

Reply:
fixed